digesta-bot/action.yml

```diff
@@ -1,127 +0,0 @@
-name: 'Update the image digest'
```

Comment:
I'm worried about breaking downstream folks.  Can we make this invoke the other action (for now) with a `::warning::` that it'll be removed on `YYYY/MM/DD` and then we delete it entirely on that date?

Reply:
sure, will update

Reply:
done, PTAL